config/android/BUILD.gn

```diff
@@ -120,6 +124,29 @@ config("runtime_library") {
                     root_build_dir),
   ]
 
+  # Unified Headers !! refer to: https://android.googlesource.com/platform/ndk/+/ndk-r15-release/docs/UnifiedHeaders.md
+  if (current_cpu == "arm") {
+      abi_target = "arm-linux-androideabi"
+    } else if (current_cpu == "x86") {
+      # no "eabi" in ndk-r16
+      abi_target = "i686-linux-android"
+    } else if (current_cpu == "arm64") {
+      # no "eabi" in ndk-r16
+      abi_target = "aarch64-linux-android"
+    } else if (current_cpu == "x64") {
+      abi_target = "x86_64-linux-android"
+  }
+
+  cflags_cc += [
+    "-isystem" +
+        rebase_path("$android_ndk_root/sysroot/usr/include",
+                    root_build_dir),
+        #only for arm32 
+    "-isystem" +
+        rebase_path("$android_ndk_root/sysroot/usr/include/$abi_target",
```

Comment:
add extra folder include, for the __Unified Headers__ changes occurs between ndk r12 and r16.

refer to: https://android.googlesource.com/platform/ndk/+/ndk-r15-release/docs/UnifiedHeaders.md